standalone/content/css/webapp.css

```diff
@@ -586,32 +586,54 @@ html[dir="rtl"] .standalone-info-bar > .media-control-buttons > .btn-hangup:afte
   font-weight: bold;
 }
 
-.room-inner-info-area > .room-waiting-area {
-  display: flex;
-  justify-content: space-between;
-  margin: 3em auto 1em;
-  /* This should match the width set in the room-waiting-tile to ensure the edges
-     of the content aligns within the tile width. */
-  width: 290px;
+.ads-tile {
+  border-top: solid 1px #c1c1c1;
+  box-sizing: border-box;
+  height: 220px;
+  padding: 0 1rem .8rem;
+  position: relative;
 }
 
-.room-inner-info-area > .room-waiting-area > a >.room-waiting-help {
-  background: transparent url("../shared/img/glyph-help-16x16.svg") no-repeat;
-  display: inline-block;
-  height: 16px;
-  margin-left: 5px;
+.ads-tile > .ads-wrapper {
+  margin: auto;
+  position: relative;
+  width: 254px;
+}
+
+.ads-tile > .ads-wrapper > p {
+  color: #808080;
+  font-size: .8rem;
+  letter-spacing: 0.7px;
+  margin: 8px 0;
+  text-transform: uppercase;
+}
+
+.ads-tile > .ads-wrapper > a > .room-waiting-help {
+  background: #ebebeb url("../shared/img/glyph-help-no-circle-16x16.svg") center center no-repeat;
+  background-size: 16px 16px;
+  border: solid 4px #fff;
+  border-radius: 50%;
+  height: 30px;
```

Comment:
I know that guidelines ask for alphabetical order, but it would be much easier to read if related constraints are together (height&width for size)